### PR TITLE
JS Modules

### PR DESCRIPTION
**Pull request by boris-lazarevic**
Introduction to JS modules

### CODE REVIEW

File: example4.mjs [hunk 1]
```diff
@@ -0,0 +1 @@
+export const example4 = () => "Ja sam funkcija";
```
Comment (NemanjaVucic) [inline example4.mjs]: @boris-lazarevic  can you pls use the English language for strings in example files?

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "version": "1.0.0",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {
+    "": {
+      "version": "1.0.0",
+      "license": "ISC"
+    }
+  }
+}
```
Comment (NemanjaVucic) [inline package-lock.json]: We don't need `package-lock-json` you can delete this file

File: package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    "type": "git",
+    "url": "git+https://github.com/NemanjaVucic/mentorship-javascript.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/NemanjaVucic/mentorship-javascript/issues"
+  },
+  "homepage": "https://github.com/NemanjaVucic/mentorship-javascript#readme"
+}
```
Comment (NemanjaVucic) [inline package.json]: We don't need `package-json` you can delete this file

**Review by NemanjaVucic — COMMENTED** [review]
Overall great job. 👏 
I have one small task for you to do and after that, we can close the topic.
Can you make folder `js-modules` and in there put all module files? index.html should stay in the root of the project

File: package-lock.json (hunk 2 above)
Comment (boris-lazarevic) [reply package-lock.json]: Should I put it into the .gitignore file?

Comment (boris-lazarevic) [reply package-lock.json]: I deleted it from the code since we don't need it.

**Review by NemanjaVucic — APPROVED** [review]

**Pull request merged**